index.js

```diff
@@ -64,22 +64,26 @@ module.exports = {
         const releaseName = `${appName}@${this.readConfig('revisionKey')}`;
 
         this.log('SENTRY: Deleting release...');
-        this.sentryCliExec(`releases delete ${releaseName}`);
+        this.sentryCliExec(`delete ${releaseName}`);
         this.log('SENTRY: Release deleted!');
       },
 
-      sentryCliExec(command) {
+      sentryCliExec(params) {
         const authToken = this.readConfig('authToken');
         const orgName = this.readConfig('orgName');
         const appName = this.readConfig('appName');
         const url = this.readConfig('url');
 
         return this._exec(
-          url ? `SENTRY_URL=${url} ` : '' +
-          `SENTRY_ORG=${orgName} ` +
-          `SENTRY_PROJECT=${appName} ` +
-          `SENTRY_AUTH_TOKEN=${authToken} ` +
-          `node_modules/.bin/sentry-cli ${command}`
+          [
+            path.join('node_modules', '.bin', 'sentry-cli'),
+            url ? `--url ${url}` : '',
+            `--auth-token ${authToken}`,
+            'releases',
```

Comment:
I preferred to have `releases` in every command. `sentryCliExec` is meant to execute sentry cli not specific releases commands. :) Could you please revert that?

Reply:
With -- parameters you can't do that.

sentry-cli has a certain order rule for that. --org and --project parameters must always come after command.

At first I was going to write like this. this.sentryCliExec('release', `new ${releaseName}`);

Two arguments first command, second params. That's why it left named params.